packages/amplify-util-mock/src/__tests__/utils/index.test.ts

```diff
@@ -1,7 +1,4 @@
-import { describe } from 'jest-circus';
```

Comment:
I just did a global search for `jest-circus` and found we have it in packages/amplify-appsync-simulator/package.json but we don't use it in that package. Maybe we can remove it. Maybe in another PR

Reply:
`jest-circus` seems to be used as test runner in couple of places. i.e. at runtime.
I'd rather not go into that whacamole game in this PR.